tests/codegen/simd-intrinsic/simd-intrinsic-generic-masked-load.rs

```diff
@@ -13,14 +14,9 @@ pub struct Vec2<T>(pub [T; 2]);
 #[derive(Copy, Clone, PartialEq, Debug)]
 pub struct Vec4<T>(pub [T; 4]);
 
-extern "rust-intrinsic" {
-    fn simd_masked_load<M, P, T>(mask: M, pointer: P, values: T) -> T;
-}
-
 // CHECK-LABEL: @load_f32x2
 #[no_mangle]
-pub unsafe fn load_f32x2(mask: Vec2<i32>, pointer: *const f32,
-                         values: Vec2<f32>) -> Vec2<f32> {
+pub unsafe fn load_f32x2(mask: Vec2<i32>, pointer: *const f32, values: Vec2<f32>) -> Vec2<f32> {
```

Comment:
There's a bunch of unrelated formatting changes as well.

If you have your editor set to format-on-save, then please turn that off. You shouldn't apply rustfmt to a file just because you change its imports a bit.

Reply:
I'm honestly surprised CI does not enforce the formatting. Why would we want poorly-formatted code? `rustfmt::skip` can be used if there is some deliberate reason?

Reply:
It's not worth the effort on tests.